src/Features/Core/Portable/EditAndContinue/DebuggingSessionTelemetry.cs

```diff
@@ -115,6 +115,15 @@ public static void Log(Data data, Action<FunctionId, LogMessage> log, Func<int>
 
                     // Ids of all projects whose binaries were successfully updated during the session.
                     map["ProjectIdsWithAppliedChanges"] = editSessionData.Committed ? editSessionData.ProjectsWithValidDelta.Select(ProjectIdToPii) : "";
+
+                    // Total milliseconds it took to emit the delta in this edit session.
+                    map["EmitDifferenceMilliseconds"] = editSessionData.EmitDifferenceTime.Milliseconds;
+
+                    // Total milliseconds it took to analyze all documents that contributed to the changes that were
+                    // attempted to be applied (whether or not the applications was successful) in this edit session.
+                    // Includes analysis that had been performed asynchronously before "apply changes" was triggered
+                    // (if we reused analysis results that were calculated by EnC analyzer for rude edit reporting).
+                    map["TotalAnalysisMilliseconds"] = editSessionData.AnalysisTime.Milliseconds;
```

Comment:
Should be `TotalMilliseconds`. `Milliseconds` is just the ms component of whatever the time is. Also applies to line 120

Reply:
Good catch. Thanks!